### ISSUE TITLE
Draft changes to Ohm's Law descriptions ready for review

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#90]
Based on user feedback this week, I have made some changes to the [Ohm's Law descriptions](https://docs.google.com/spreadsheets/d/15LuKXlzkh6xHpNbWC4ujtOfz1F7FJ7tj4oUF8Jvmj-k/edit#gid=0).

Would anyone like to comment on the changes. The changed descriptions are in the first column (orange column).

@emily-phet, @arouinfar, @oliver-phet, @jessegreenberg, @zepumph 

You can comment here, or in the document. I find commenting on google sheets to be a little cumbersome.

**Comment by jessegreenberg** [comment]
@terracoda everything looks good to me!

I noticed you changed  summaries to sentences instead of lists. Is that because of https://github.com/phetsims/ohms-law/issues/82, or do you believe that is a better experience? If because of https://github.com/phetsims/ohms-law/issues/82, we can probably prioritize fixing that issue before this sim is deployed with a11y (and we will probably have to if we want to use ```<em>``` and ```<strong>``` tags.)

**Comment by terracoda** [comment]
@jessegreenberg, I did remove the list, partly due to #82 , but also exploring a briefer representation. I will look now at the design doc to see how the new descriptions work while keeping the lists. I's actually like to know in general is users prefer getting little lists, or a more paragraph like description.

**Comment by terracoda** [comment]
@jessegreenberg, I added both options, list structure and paragraph structure, so we can potentially more easily update the strings without changing the current structure. 

**Note** I also changes to curly braces as that's what you use in the a11y strings file - keeps things simple for copying.

**Comment by arouinfar** [comment]
Looks good @terracoda!

B19 & B22:
> Question: Should we refer to "current" as "current, I" in the alert?

I don't think it's necessary here, and I think it'd look a bit unbalanced unless you also say "voltage, V" and "resistance, R".


B22: 
> Question: Should add "a lot" to communicate large size changes for I?

I like this suggestion! Since R and I have an inverse relationship, users can definitely encounter bigger changes, so I think using the "a lot" modifier when jumping 2+ sizes may help cue the differences between the V/I and R/I relationships.

**Comment by terracoda** [comment]
@ariel-phet, @arouinfar thinks the updates are good. Could you have a read through of column B (the orange column) in the google sheet. I've marked content changes in red. You can see where content has been reduced by comparing with the orange column (column B) with the green column (column C).

**Comment by terracoda** [comment]
@emily-phet and @ariel-phet, do either of you want to read through the changes to Ohm's Law descriptions?

I'd like to ask @jessegreenberg or @zepumph update the current descriptions.

**Comment by zepumph** [comment]
Just so everyone is aware of the current plan. Once this issue has been reviewed by @ariel-phet & co and implemented by @jessegreenberg. I will go ahead with #92 so that the sonification wrapper can use a new dev version that has updated descriptions for @terracoda to use in interviews hopefully mid next week.

**Comment by ariel-phet** [comment]
@terracoda a few comments

B4: Why did your remove the equation description? Isn't knowning that V=I*R is displayed important (since there are several other ways one could algebraically rearrange this equation...or if someone does not remember the equation?) Is it because the student explores to B12 and that clears it up?

B12: yes, that is the best order IMO

B14: "Batteries total" is kind of a strange phrasing. Perhaps "Batteries outputting" or something along those lines? "Batteries providing" "Batteries supplying" (I think "Batteries supplying" is best)

B19: I think just "current" is fine

**Comment by terracoda** [comment]
@ariel-phet, awesome feedback! Thank-you. 

B4: I removed the actual equation from the Scene Summary to simplify the set up, however, I think it can stay. One repetition of the actual equation is likely not a bad thing.

B14: Great suggestions for the batteries. I was thinking of them representing a total voltage, but you are correct, batteries **supply** voltage.

B19: We'll keep **I** out of the alert and see how more users make sense of it. I think the other simplifications with set things up more directly.

**Comment by terracoda** [comment]
@jessegreenberg and @zepumph I've updated the Ohm's Law description spreadsheet. The new descriptions are ready for implementation. 

We can try again with the list structure to see how the simplified content sounds. I think there is at least one less list now.

If we cannot resolve #82 easily, I think I think it would be a good idea replace the lists with the paragraphs in this sim. There is very little content in general in this sim, so I think the paragraphs will read fine.

Let me know if you have any questions.

**Comment by zepumph** [comment]
I'll take a look at this. I am going to keep the lists for now, and we can always change them pretty easily if #82 doesn't pan out.

**Comment by zepumph** [comment]
I implemented everything exactly as it is in the orange column with a few exceptions/questions:

* In scene summary, each `<li>` after "Right now" is capitalized. I'm happy to make it lower case, I just wanted to make sure we care first. Also know that the list under "The Circuit" in the play area does begin lower case. 
* In the doc it looks like you expect the default to be "much larger than" under "Ohm's Law Equation". From the doc: "In equation, letter V is {{much larger than}} letter I and {{comparable to}} letter R."
But it is actually "much much larger than". How do we feel about that? I haven't been too involved in this design process, so I'm not sure what is best, but to me it doesn't seem like it should start in the most extreme comparative case.

**Comment by terracoda** [comment]
Thanks @zepumph ,
- lower case on the bullets is fine. I thin I had then capitalized before as I had the lists punctuated as sentences. They re no longer punctuated that way. Yes, we care about capitalization and punctuation (I do not always get it right).
- I was guessing at the letter size, so we can go with the model and see how that sounds and looks.
- I think it is ok, that the the beginning of the circuit description is one paragraph that breaks at the beginning of the list. Did you do the same thing for the 2 sentences describing the equation (no list there, though)?

**Comment by zepumph** [comment]
>Did you do the same thing for the 2 sentences describing the equation (no list there, though)?

Not quite, there we have the first p as the accessibleDescription to the whole FormulaNode ("Voltage, V, is equal to Current, I, . . ."), and then another 'p' tag is a child of that node which holds the dynamic 'In equation, letter V. . . '


Update: changed above, not accessibleLabel but accessibleDescription

**Comment by zepumph** [comment]
@terracoda I'm confused about another box B19. I see that some of the dynamic pieces are italicized.

 {{3.6 volts.}} As letter V _{{shrinks}}_, letter I _{{shrinks}}_. Current now _{{7.2 millamps}}_ with voltage at _{{3.6 volts}}_.

Is this deliberate? Part of my confusion is that the curly braces are italicized also, perhaps just meaning that you are emphasizing that this value is dynamic?

Currently we only support these html formatting tags in descriptions and not in alerts. I think that eventually that would be a good thing to add to the mixin, but we aren't quite there yet. @jessegreenberg how long would you expect it to take to support html formatting tags in alerts if we go down that road?

**Comment by jessegreenberg** [comment]
@zepumph I would guess 2-4 hours, we would need to make the changes to AriaHerald/UtteranceQueue/Utterance doing similar things as were done in the Accessibility mixin.

**Comment by terracoda** [comment]
@zepumph, yes the emphasis was to highlight that they are dynamic. You are correct, however, these are alerts! It is **not so important to highlight** and I do not want to complicate any live region content!

I do not think we need formatted content in the alerts.

**Comment by zepumph** [comment]
All other changes have been pushed to master

**Comment by jessegreenberg** [comment]
> I do not think we need formatted content in the alerts.

Thanks @terracoda, we won't add formatting to alerts until it is requested.

**Comment by zepumph** [comment]
Awesome!!! @terracoda thanks for all the work on these. I have updated the descriptions on master, and you can get them here or phettest. 
http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.14/ohms-law-a11y-view.html

I have also published a new phetio version and hooked it into the sonification wrapper, then I deployed that.

http://www.colorado.edu/physics/phet/dev/html/phet-io-wrapper-sonification/1.0.0-dev.27/phet-io-wrapper-sonification/sonification.html

You should be all set to use updated descriptions in future interviews. Let me know if there is more to do!

**Comment by emily-phet** [comment]
@terracoda @zepumph This may have been addressed in a newer version, but when playing with th e1.4.0-dev.14 linked above, the circuit arrow description stays at "tiny arrow" for the vast majority of cases. It's only when the arrow visually appears quite large that this "tiny arrow" description changes. Feels incorrect to me.

**Comment by terracoda** [comment]
@zepumph, it looks the descriptions changes more, but the size description still doesn't match what I think it should be visually.

**Comment by terracoda** [comment]
@zepumph re my last comment, would it be possible to map out the 7 sizes visually, and see how that fits with the model. It's possible that the ranges for each size are not even.

**Comment by terracoda** [comment]
@zepumph and @jessegreenberg, I think all the descriptions in this issue have been implemented. So I will close it. 

Based on subsequent user comments, I think it is safe to shorten the alert by removing the repeated slider value. I will start a new issue for that.
